extensions/ohif-cornerstone-extension/src/commandsModule.js

```diff
@@ -91,59 +91,38 @@ const actions = {
     console.warn('updateDisplaySet: ', direction);
   },
   clearAnnotations: ({ viewports }) => {
-    const element = _getActiveViewportEnabledElement(
-      viewports.viewportSpecificData,
-      viewports.activeViewportIndex
-    );
-    if (!element) {
-      return;
-    }
+    const enabledElement = _getEnabledElement(viewports);
+    const enabledElementToolState = _getElementToolState(enabledElement);
 
-    const enabledElement = cornerstone.getEnabledElement(element);
-    if (!enabledElement || !enabledElement.image) {
-      return;
-    }
+    if (enabledElementToolState) {
+      Object.entries(enabledElementToolState).forEach(
```

Comment:
`Object.entries` needs a polyfill, which we add in our app. Perhaps OHIF may need to poyfill this too if they haven't so.

Reply:
Object.entries is used in ohif already - https://github.com/OHIF/Viewers/blob/master/src/studylist/StudyListRouting.js#L17
And it looks like they've got a polyfill in their yarn.lock - https://github.com/OHIF/Viewers/blob/master/yarn.lock#L10436

Not sure why it's not also in the package.json.